lib/Container/Compiler/LayoutsPageIndexingPass.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Netgen\IbexaSearchExtra\Container\Compiler;
+
+use Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\DocumentMapper\DocumentFactory;
+use Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\Query\CriterionVisitor\Content\VisibilityVisitor as ContentVisibilityVisitor;
+use Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\Query\CriterionVisitor\Location\VisibilityVisitor as LocationVisibilityVisitor;
+
+use Netgen\IbexaSearchExtra\Core\Search\Solr\FieldMapper\ContentTranslation\ContentPageTextFieldMapper;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Ibexa\Elasticsearch\DocumentMapper\DocumentFactoryInterface;
+use Ibexa\Contracts\Core\Persistence\Content\Handler;
+
+
+class LayoutsPageIndexingPass implements CompilerPassInterface
```

Comment:
Rename to `PageIndexingPass`. This is not Layouts specific, so we should remove all mention of "layouts" in the PR.

Reply:
resolved in https://github.com/netgen/ibexa-search-extra/commit/497a329387786154c853232148c58bd86e3e230c